charts/index.yaml

```diff
@@ -0,0 +1,220 @@
+apiVersion: v1
```

Comment:
is this file auto-generated? what if we build a new version or new chart?

Reply:
this is generated by the command `helm repo index charts --url https://open-cluster-management.io/helm-charts/`, I think we should add a new workflow to auto-generate it.

Reply:
Let's merge this first to test integrating with artifactHub, then I will fix the comment with a follow-up PR.

Reply:
ok, this is a followup we should do , right?